ReactWindows/ReactNative.Common/Reflection/ReflectionHelpers.cs

```diff
@@ -19,7 +19,7 @@ static class ReflectionHelpers
         /// </returns>
         public static bool IsAsync(this MethodInfo methodInfo)
         {
```

Comment:
Delete this method, we're not going to use it afterall.  At one point, I had considered adding functionality where [ReactMethod]s could be implemented to return Task instead of using the IPromise, but I think people would make mistakes like not using ConfigureAwait(false) and having a lot of issues with thread stealing.